tests/app/conftest.py

```diff
@@ -2405,7 +1816,7 @@ def _sample_inbound_number(
 
 @pytest.fixture
 def sample_inbound_numbers(sample_service, sample_inbound_number):
-    service = sample_service(service_name=str(uuid4()), check_if_service_exists=True)
+    service = sample_service(service_name=str(uuid4()), check_if_service_exists=False)
```

Comment:
Curious why we updated this - I'm trying to understand the code. 

Not saying this is wrong or anything needs to be correct.